influxql/ast.go

```diff
@@ -598,7 +598,7 @@ type SelectStatement struct {
 	groupByInterval time.Duration
 
 	// if it's a query for raw data values (i.e. not an aggregate)
-	RawQuery bool
+	IsRawQuery bool
```

Comment:
I prefer `IsRawQuery` too. `RawQuery` makes me think that it's a `Query` type if I saw it used somewhere.